site/content/docs/browser-checks/_index.md

```diff
@@ -9,22 +9,43 @@ aliases:
     - /docs/browser-checks/quickstart/
 ---
 
+This quick start gives you all the info to create your first Browser check with Checkly. You should have some prior
+knowledge of working with Javascript and/or Node.js.
 
+## What is a Browser check?
 
-This quick start should give you all the info to create your first browser check with Checkly. You should have some prior
-knowledge working with Javascript and/or Node.js.
+A Browser check is a Node.js script that controls a Chrome browser to mimic user behavior.
+Load a web page, click a link, fill a form input – do everything your visitors might do and check if these interactions lead to the correct results.
 
-## What is a browser check?
+Your critical interactions might be:
 
-The five lines of code below are already a valid browser check. Its usefulness might be not that great, but still...
+- That the shopping cart is visible.
+- That users can add products to the shopping cart.
+- That users can log into my app.
+
+Use Browser checks to guarantee that your site works as expected.
+
+Checkly uses the **[Playwright](https://github.com/microsoft/playwright)** and **[Puppeteer](https://github.com/GoogleChrome/puppeteer)**
+frameworks to power your Browser checks. Use these frameworks to control the interactions you want to happen on a web page.
+
+{{< info >}}
+While Playwright and Puppeteer share many similarities, they have evolved at different speeds over the last two years. Playwright's rapid release cycle and new  features such as [auto-waiting](https://playwright.dev/docs/actionability) and [the built-in inspector](https://playwright.dev/docs/debug#playwright-inspector) made it gain momentum in the developer community.
```

Comment:
I would be even bolder here and add "We strongly recommend using Playwright if you are just starting out."

Additional idea would be linking the migration script, too: "If you would like to migrate from Puppeteer to Playwright, you can use our [migration script](https://github.com/checkly/puppeteer-to-playwright)."

Reply:
Nice. I like that!